server/seederinit.js

```diff
@@ -1,3 +1,8 @@
 const seeder = require('./seeder.js');
 
-seeder.seedDatabase(100, 200);
+const min = 100;
+const max = 300;
```

Comment:
It would be nice if these were specified by environment variables, so you don't need to modify the code to chance the range of item ids.